pallets/ajuna-awesome-avatars/benchmarking/src/lib.rs

```diff
@@ -77,7 +77,8 @@ benchmarks! {
 
 		let season = Seasons::<T>::get(CurrentSeasonStatus::<T>::get().season_id).unwrap();
 		let mint_fee = season.fee.mint.fee_for(&MintPackSize::Six);
-		CurrencyOf::<T>::make_free_balance_be(&caller, mint_fee);
+		let ed = CurrencyOf::<T>::minimum_balance();
+		CurrencyOf::<T>::make_free_balance_be(&caller, mint_fee + ed);
```

Comment:
The fee handler errors if the account would be reaped (which is a new behavior), this affected the outcomes of these tests that were missing the funds.